_includes/manuals/1.17/4.4.2_prov_media.md

```diff
@@ -1,5 +1,5 @@
 
-The Installation Media represents the web URL from where the installation packages can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when Foreman is first installed. However, it is best to edit the media you are going to use and ensure the Family is set.
+The Installation Media represents the web URL from where the installation packages can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when Foreman is first installed. However, it is best to edit the media you are going to use and ensure the Family is set. In the Red Hat World, installation media are a resource for the installation repo.
```

Comment:
This addition doesn't clarify it much for me.

Reply:
Yeah I'd drop this.

Reply:
@lzap, the clarification was taken verbatim from your own suggestion in https://community.theforeman.org/t/need-use-case-for-different-installation-media-for-single-operating-system/8320, and it doesn't clarify much for me, but you said it was important to understand, so I put the text in my patch.